modules/ROOT/pages/retrieve-unified-schema.adoc

```diff
@@ -0,0 +1,23 @@
+= Retrieve a Copy of the Unified Schema
+
+You can get a copy of the unified schema using a third-party API client. 
```

Comment:
```suggestion
You can get a copy of the unified schema by using a third-party API client. 
```
...because the schema is not what's using the client, right?